docs/csharp/whats-new/csharp-version-history.md

```diff
@@ -7,7 +7,19 @@ ms.date: 09/20/2017
 
 # The history of C# #
 
-What did the language look like in its earliest incarnations? And how has it evolved in the years since?
+This article provides a history of each major release of the C# language. The C# team is continuing to innovate and add new features. Detailed language feature status, including features considered for upcoming releases can be found [on the dotnet/roslyn repository](https://github.com/dotnet/roslyn/blob/master/docs/Language%20Feature%20Status.md) on GitHub.
+
+> [!IMPORTANT]
+> The C# language relies on types and methods in a *standard library* for some of the features. One example is exception processing. Every `throw` statement or expression is checked to ensure the object being thrown is derived from <xref:System.Exception>. Similarly, every `catch` is checked to ensure that the type being caught is derived from <xref:System.Exception>. Each version may add new requirements. To use the latest language features in older environments, you may need to install specific libraries. These dependencies are documented in the page for each specific version. You can learn more about the [relationships between language and library](relationships-between-language-and-library.md) for background on this dependency.
+
+The C# build tools consider the latest major language release the default language version. There have been point releases since the last major release, detailed in other articles in this section. To use the latest features in a point release, you need to [configure the compiler language version](../language-reference/configure-language-version.md) and select the version. There have been three point releases since C# 7.0:
```

Comment:
There have been point releases --> This is an absolute statement that may not always be true for the latest release and so requires deletion for major releases. Do you want to qualify it ("There may be point releases...") Otherwise, the rest of the paragraph can remain unchanged.